github-for-jira.sd.yml

```diff
@@ -313,6 +313,7 @@ environmentOverrides:
         attributes:
           parameters:
             DBInstanceClass: db.t3.large
+            AllocatedStorage: 10 # 10 GB
```

Comment:
Do we need this for all environments or just for production?

Reply:
Just production should be fine.